ui/imports/shared/views/chat/LinksMessageView.qml

```diff
@@ -272,16 +274,49 @@ Column {
                 isCurrentUser: root.isCurrentUser
                 anchors.horizontalCenter: parent.horizontalCenter
                 anchors.top: parent.top
-                playing: root.messageStore.playAnimation
                 isOnline: root.store.mainModuleInst.isOnline
-            }
+                onClicked: {
+                    if (linkData.thumbnailUrl.indexOf(".gif") > -1) {
```

Comment:
And extract it somewhere, this is repeated all over the place

Reply:
done!